jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/WebAppContextTest.java

```diff
@@ -848,9 +848,13 @@ public void testRestartWebApp(WorkDir workDir) throws Exception
         assertThat("URLs (sub) refs", actualRefs, containsInAnyOrder(expectedRefs));
 
         // Simulate a reload
+        LOG.info("Stopping Initial Context");
         context.stop();
-        Thread.sleep(1000);
+        LOG.info("Stopped Initial Context - waiting 2 seconds");
+        Thread.sleep(2000);
```

Comment:
Can't `touch()` set the file's timestamp 2s in the future instead of sleeping here?

Reply:
I can give that a try.